eslint.config.js

```diff
@@ -16,6 +16,7 @@ export default [
 			'docs/examples/geojson/sample-geojson.js',
 			'docs/examples/map-panes/eu-countries.js',
 			'docs/examples/extending/extending-2-layers.md',
+			'docs/examples/quick-start/index.md',
```

Comment:
Why is this one excluded?

Reply:
Eslint is throwing an lint error for `"imports":` in the importmap. I wasn't able to ignore only this specific error for this line / file or fix it with updating eslint.
```
<script type="importmap">
	{
		"imports": {
			"leaflet": "https://unpkg.com/leaflet@{{ site.latest_leaflet_version}}/dist/leaflet.js"
		},
```

Reply:
Good enough a reason for me